rholang/src/test/scala/coop/rchain/rholang/interpreter/accounting/CostAccountingPropertyTest.scala

```diff
@@ -77,11 +77,12 @@ class CostAccountingPropertyTest extends AnyFlatSpec with ScalaCheckPropertyChec
 
 object CostAccountingPropertyTest {
 
-  def haveEqualResults[A](tasks: Task[A]*)(implicit duration: Duration): Boolean =
+  def haveEqualResults[A](tasks: IO[A]*)(implicit duration: FiniteDuration): Boolean =
     tasks.toList
-      .sequence[Task, A]
+      .sequence[IO, A]
       .map { _.sliding(2).forall { case List(r1, r2) => r1 == r2 } }
-      .runSyncUnsafe(duration)
+      .unsafeRunTimed(duration)
+      .get
```

Comment:
Let's use this pattern when we need sync value and timeout exception. It's better then None.get exception.
```scala
.timeout(duration).unsafeRunSync()
```